pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/ClusterTest.java

```diff
@@ -196,6 +217,24 @@ protected void startServers(int numServers, PinotConfiguration configuration, in
     }
   }
 
+  protected void startServerHttps() {
+    FileUtils.deleteQuietly(new File(Server.DEFAULT_INSTANCE_BASE_DIR));
+    _serverStarters = new ArrayList<>();
+
+    Map<String, Object> properties = getDefaultServerConfiguration().toMap();
+
+    PinotConfiguration configuration = new PinotConfiguration(properties);
+    overrideServerConf(configuration);
```

Comment:
Do we need to set extra properties like `startServers()`?

Reply:
turns out the test code already sets default values for pinot server in involved ways. imo pinot's test code should be cleaned up, but that's another story